src/core/config.rs

```diff
@@ -46,24 +43,37 @@ pub struct Config {
 	pub num_hops: u32,
 	/// Average number of seconds to delay each each message fragment at each hop.
 	pub average_message_delay_ms: u32,
+	/// Limit number of message in a windows of time for a peer.
+	/// Default value, this can be change from topology.
+	/// Above limit message are drop, so topology should raise the
+	/// limit for routing peers.
+	/// `None` is unlimited.
+	/// Window is `WINDOW_BACKPRESSURE` duration.
+	pub limit_per_window: Option<u32>,
```

Comment:
`target_bits_per_second` was supposed to limit incoming traffic rate as well. Do we really need a separate config?